rgw/v2/tests/s3_swift/test_bucket_lifecycle_object_expiration.py

```diff
@@ -54,8 +54,14 @@ def test_exec(config):
     io_info_initialize.initialize(basic_io_structure.initial())
     ceph_conf = CephConfOp()
     rgw_service = RGWService()
-    config.rgw_lc_debug_interval = 30
-    config.rgw_lc_max_worker = 10
+    buckets = []
+    config.rgw_lc_debug_interval = (
+        config.rgw_lc_debug_interval if config.rgw_lc_debug_interval else 30
+    )
+    config.rgw_lc_max_worker = (
+        config.rgw_lc_max_worker if config.rgw_lc_max_worker else 10
+    )
+    config.parallel_lc = config.parallel_lc if config.parallel_lc else False
```

Comment:
Can we mention default values in resource.op.py file

Reply:
done